src/app/tables/artifacts-table/artifacts-table.component.ts

```diff
@@ -80,6 +80,14 @@ export class ArtifactsTableComponent implements OnChanges {
         'See Story': this.getNodeStoryUrl(artifact)
       };
 
+      // If there are associated advisories, we can add links to those as well.
+      if (this.defaultColumns.indexOf('Advisories') > -1 && artifact.hasOwnProperty('advisories')) {
+        if (artifact.advisories[0]) {
+          const url = externalUrlPipe.transform(artifact.advisories[0]);
+          this.linkColumnMappings[formattedArtifact[this.uidColumn]]['Advisories'] = url;
```

Comment:
This would make the comma separated list of advisories link to the first advisory. This is not very intuitive. We could either drop this requirement or you could find a way to support multiple links, though the template in `table.component.html` is already fairly complex. See what you can come up with and if it gets too complicated, we can drop this requirement.